python/feldera/rest/connector.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any, Mapping, Optional
+from feldera.rest.attached_connector import AttachedConnector
+
+
+class Connector:
+    """
+    A generic connector class that can be used to represent any Feldera connector
+    """
+    name: str
+    description: Optional[str]
+    config: Mapping[str, Any]
+    id: Optional[str]
+
+    def __init__(
```

Comment:
Additional parameter doc here is useful as it is user-facing when creating a connector (as for the other objects).